lua/lv-vimtex/init.lua

```diff
@@ -23,3 +22,7 @@ vim.api.nvim_exec(
     ]],
   false
 )
+if (O.vimtex_autosave)
```

Comment:
can you add a `auto_save = false` inside [lua/default-config.lua](https://github.com/ChristianChiarulli/LunarVim/blob/rolling/lua/default-config.lua#L151) 
and use `O.lang.latex.auto_save` instead of `O.vimtex_autosave`

Reply:
Thank you for your comments, I think its much cleaner now.